internals/maintenance_scripts.py

```diff
@@ -576,3 +576,51 @@ def get_template_data(self, **kwargs):
       stage.put()
 
     return f'{len(ot_stages)} trial creation request(s) processed.'
+
+
+class ActivateOriginTrials(FlaskHandler):
+
+  def _get_today(self):
+    return date.today()
+
+  def get_template_data(self, **kwargs):
+    """Activate any origin trials that are flagged for activation."""
+    self.require_cron_header()
+
+    success_count, fail_count = 0, 0
+    today = self._get_today()
+    # Get all OT stages.
+    ot_stages: list[Stage] = Stage.query(
+        ndb.OR(
```

Comment:
Why not use the IN operator?

Reply:
I didn't think to use it, but I think it reads better anyway. 🙂 Interestingly, after reading some more, it seems [using the IN and OR operators are functionally identical](https://cloud.google.com/datastore/docs/concepts/queries#in). I still think it's easier to interpret the meaning using the IN operator.